src/engine/sparqlExpressions/SparqlExpressionValueGetters.cpp

```diff
@@ -119,8 +119,8 @@ std::optional<std::string> StringValueGetter::operator()(
 std::optional<ad_utility::triple_component::Literal>
 LiteralValueGetterWithStrFunction::operator()(
     Id id, const EvaluationContext* context) const {
-  return ExportQueryExecutionTrees::idToLiteral(context->_qec.getIndex(), id,
-                                                context->_localVocab);
+  return ExportQueryExecutionTrees::idToLiteral(
+      context->_qec.getIndex().getImpl(), id, context->_localVocab);
```

Comment:
This should be an additional overload, instead of having to update all the call sites.